weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -1282,6 +1312,7 @@ def _ch_call_to_call_schema(ch_call: SelectableCHCallSchema) -> tsi.CallSchema:
         exception=ch_call.exception,
         wb_run_id=ch_call.wb_run_id,
         wb_user_id=ch_call.wb_user_id,
+        display_name=ch_call.display_name[1] if ch_call.display_name else None,
```

Comment:
even further, the `[1]` indexing should be done in the db layer. we want to be able to select `display_name` directly, hiding the implementation details as to how that is achieved.